recipe/meta.yaml

```diff
@@ -1,44 +1,59 @@
-{% set name = "trio-asyncio" %}
+{% set name = "trio_asyncio" %}
```

Comment:
Not to bikeshed, but if it's just a matter of picking one, I think it makes more sense to keep the main output as the dashed version to match the feedstock name, and then add the other as an alias.

Otherwise, we need the `extra.feedstock-name` correction.

Reply:
Fair enough. I mainly did it that way to use the name template in the url but I can just hard-code it there